tools/nni_trial_tool/trial_keeper.py

```diff
@@ -118,13 +121,20 @@ def check_version(args):
         try:
             trial_keeper_version = regular.search(trial_keeper_version).group('version')
             nni_log(LogType.Info, 'trial_keeper_version is {0}'.format(trial_keeper_version))
-            training_service_version = regular.search(args.version).group('version')
-            nni_log(LogType.Info, 'training_service_version is {0}'.format(training_service_version))
-            if trial_keeper_version != training_service_version:
+            nni_manager_version = regular.search(args.version).group('version')
```

Comment:
suggest to change args.version to args.nni_manager_version.

Reply:
changed to `nni_manager_version`